### PR TITLE
Release 0.8.1 part 1/3: CHANGELOG, README, Makefile

### PR DESCRIPTION
**Pull request by khoang98**
**Is this a bug fix or adding new feature?**
Neither

**What is this PR about? / Why do we need it?**
Updating AL2 base image

**What testing is done?** 
None

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -121,7 +124,7 @@ kubectl apply -f secret.yaml
 
 #### Deploy driver
 ```sh
-kubectl apply -k "github.com/kubernetes-sigs/aws-fsx-csi-driver/deploy/kubernetes/overlays/stable/?ref=release-0.8"
+kubectl apply -k "github.com/kubernetes-sigs/aws-fsx-csi-driver/deploy/kubernetes/overlays/stable/?ref=release-0.8.1"
```
Comment (wongma7) [inline docs/README.md:127]: this can stay release-0.8 (it's a minor version branch)

Comment (khoang98) [reply docs/README.md:127]: Ack, will change

**Pull request merged**